### PR TITLE
Production deploy script [SATURN-1364]

### PR DESCRIPTION
**Pull request by petesantos**
Adds a production deploy script for manual deploys

### CODE REVIEW

File: scripts/prod-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eo pipefail
+yarn install
+yarn lint
+yarn generate-docs
+cp config/prod.json config.json
+gcloud app deploy --project=terra-bard --promote --quiet
```
Comment (panentheos) [inline scripts/prod-deploy.sh]: This should be `terra-bard-prod`, right?

File: prod.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "project": "terra-bard-prod",
+  "samRoot": "https://sam.dsde-prod.broadinstitute.org"
+}
```
Comment (cameron-ardell) [inline prod.json]: since this code is a repeat of `config.json`, do we want to do a symbolic link instead?

File: config/prod.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
-  "project": "terra-bard-prod",
+  "project": "terra-bard",
   "samRoot": "https://sam.dsde-prod.broadinstitute.org"
```
Comment (cameron-ardell) [inline config/prod.json]: since this file is the same as `config.json`, do we want to do a symbolic link instead?

Comment (petesantos) [reply config/prod.json]: config.json was accidentally added - thanks for pointing this out!

**Comment by cameron-ardell** [comment]
I don't feel like I know enough about the script to evaluate it fully, but looks good to me!
I'd feel more comfortable with someone else giving this a thumb beside me though given that.

**Review by breilly2 — APPROVED** [review]
Looks good. I did not test manually, though I assume that you did. I did compare this to the deploy scripts for lyle and calhoun and this looks like it's doing the right thing.

File: prod.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
```
[the remark this answers is not in the document]
Comment (breilly2) [reply prod.json]: Never mind this. It's from an old version of this PR.

**Pull request merged**